src/bin/commands/parse-docs.ts

```diff
@@ -0,0 +1,27 @@
+import { readFile, writeFile } from 'fs/promises';
+import ora from 'ora-classic';
+import { resolve } from 'path';
+import type { JSONOutput } from 'typedoc';
+import { ProjectParser } from '../../lib/structures/ProjectParser';
+import type { Options } from '../lib/types/Options';
+
+export async function parseDocs(options: Options) {
+  const spinner = ora('Parsing TypeDoc JSON output').start();
+
+  const project = JSON.parse(await readFile(resolve(process.cwd(), options.json), 'utf-8')) as JSONOutput.ProjectReflection;
+  const parsed = new ProjectParser(project).toJSON();
+
+  try {
+    await writeFile(resolve(process.cwd(), options.json), JSON.stringify(parsed, null, 2));
```

Comment:
As you're reading from and writing to the same file you should preserve indents and newline characters. You can yoink https://github.com/favware/cliff-jumper/blob/main/src/lib/package-json-parser.ts for that, I myself yoinked it from NPM's code.

Reply:
Not exactly necessary here. These JSON files are only read from for `TypeDoc`'s output and then parsed and wrote with my json output